llvmlite/binding/passmanagers.py

```diff
@@ -112,6 +120,82 @@ def add_dead_arg_elimination_pass(self):
         """See http://llvm.org/docs/Passes.html#deadargelim-dead-argument-elimination."""  # noqa E501
         ffi.lib.LLVMPY_AddDeadArgEliminationPass(self)
 
+    def add_dependence_analysis_pass(self):
+        """See https://llvm.org/docs/Passes.html#da-dependence-analysis"""  # noqa E501
+        ffi.lib.LLVMPY_AddDependenceAnalysisPass(self)
+
+    def add_dot_call_graph_pass(self):
+        """See https://llvm.org/docs/Passes.html#dot-callgraph-print-call-graph-to-dot-file"""  # noqa E501
+        ffi.lib.LLVMPY_AddCallGraphDOTPrinterPass(self)
+
+    def add_dot_cfg_printer_pass(self):
```

Comment:
Should this also match the FFI name?

Reply:
It looks like this is adding the callgraph viewer (``dot-callgaph`) pass, and not the CFG viewer pass (`dot-cfg`) - is this intended?